alembic/env.py

```diff
@@ -0,0 +1,76 @@
+from logging.config import fileConfig
+
+from alembic import context
+from sqlalchemy import engine_from_config
+from sqlalchemy import pool
+
+from api.core.database import metadata
+from api.core.settings import settings
+import api.core.database.models
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
+config = context.config
+
+# Interpret the config file for Python logging.
+# This line sets up loggers basically.
+fileConfig(config.config_file_name)
+
+
+
+target_metadata = metadata
+
+
+config.set_main_option("sqlalchemy.url", settings.database_url)
```

Comment:
1. There is no need for setting this and is just being "extra", we can directly just use this variable `setttings.database_url` when performing the migrations below.

2. This is a could do, you can "fix" the spacing between the lines.

```suggestion
fileConfig(config.config_file_name)

target_metadata = metadata
config.set_main_option("sqlalchemy.url", settings.database_url)
```

Reply:
Sure 👍🏻 

Since Ves is on vacation (iirc), @jchristgit could you provide your opinions on the docker image?